template/contracts/src/WhitelistNFT.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Can you also update the contracts `Readme` with what the contract does for anyone new?

Reply:
yup!

Reply:
Thank you!